### PR TITLE
Project filter

### PR DESCRIPTION
**Pull request by bromagosa**

### CODE REVIEW

**Comment by cycomachead** [comment]
Thanks for doing the magnifying glass! This looks great! :)

**Comment by cycomachead** [comment]
Minor bug I noticed: The scrollbar doesn't update as the list is filtered.

File: blocks.js [hunk 1]
```diff
@@ -10146,6 +10149,48 @@ SymbolMorph.prototype.drawSymbolRobot = function (canvas, color) {
     return canvas;
 };
 
+SymbolMorph.prototype.drawSymbolMagnifyingGlass = function (canvas, color) {
```
Comment (jmoenig) [inline blocks.js]: this is beautiful, Bernat!

File: morphic.js [hunk 2]
```diff
@@ -1888,6 +1888,14 @@ Color.prototype.dansDarker = function () {
     return result;
 };
 
+Color.prototype.inverted = function () {
```
Comment (jmoenig) [inline morphic.js]: yep, good idea!

**Comment by jmoenig** [comment]
Thanks, @bromagosa , this is really nice! I'll pull this asap when I get around to it

**Comment by BookOwl** [comment]
"asap when I get around to it" is kind of contradictory. :P

**Comment by bromagosa** [comment]
Glad you liked it! :)

**Pull request merged**